src/components/invest/index.js

```diff
@@ -375,7 +375,7 @@ export class Invest extends React.Component {
             <a href="https://kovan.etherscan.io/tx/0x42073576a160206e61b4d9b70b436359b8d220f8b88c7c272c77023513c62c3d">transaction</a> on Kovan network.
           </p>
         </div>
-        <div className="invest-table-cell invest-table-cell_right">
+        <div className={`invest-table-cell invest-table-cell_right${investThrough === INVESTMENT_OPTIONS.QR ? ' qr-selected': ''}`}>
```

Comment:
We should consider adding https://github.com/JedWatson/classnames

Reply:
Thanks for the suggestion, just added the utility